src/client/unity.yml

```diff
@@ -0,0 +1,280 @@
+openapi: "3.0.0"
+info:
+  title: Unity API Service
+  version: 0.1.0
+servers:
+  - url: /unity/v2 # to be determined
```

Comment:
Maybe simply `/ui/v2`.

Reply:
When this moves to Quartz, it will be `/api/v2private`.